packages/vue/src/components/FSortFilterDataset/FSortFilterSorter.ts

```diff
@@ -33,15 +38,17 @@ function compare<T>(
         return fixOrder(booleanCompare(value1, value2), ascending);
     }
 
-    if (typeof value1 !== typeof value2) {
-        throw Error(
-            `Sorting is not supported for rows containing different types (attribute '${attribute.toString()}')`,
-        );
+    // We have mixed types, then we assume that the value of type string is incorrect (parser has not been applied after validation error).
+    // Values with error should always be sorted first.
+    if (typeof value1 === "string") {
+        return -1;
+    } else {
+        return 1;
     }
+}
 
-    throw Error(
-        `Sorting is only supported for types number, string and boolean (attribute '${attribute.toString()}')`,
-    );
+function isSupportedType<T>(value: T): boolean {
```

Comment:
["Golden Rule of Generics"](https://microsoft.github.io/TypeScript-New-Handbook/everything/#type-parameters-should-appear-twice)

> "Type Parameters Should Appear Twice"

```suggestion
function isSupportedType(value: unknown): boolean {
```